src/advert_phone_numbers.py

```diff
@@ -0,0 +1,38 @@
+from core_data_modules.util import PhoneNumberUuidTable
+from core_data_modules.logging import Logger
+
+from src.lib.pipeline_configuration import PipelineConfiguration
+
+import csv
+
+log = Logger(__name__)
+
+class AdvertPhoneNumbers(object):
+    @staticmethod
+    def generate(data, phone_number_uuid_table, advert_phone_numbers_csv_output_path):
+        advert_phone_numbers = set()
+
+        '''
+        Generates a csv file with normalised phone numbers for respondents who sent messages 
+        that were not labelled as Noise_Other_Project.
+
+        :param data:TracedData objects that have been manually labelled. 
+        :type: TracedData
+        :param: phone_number_uuid_table
+        :type: a look up table containing uuids to retrieve phone numbers from.
```

Comment:
core_data_modules.PhoneNumberUuidTable (or something like that at least, going from memory)

Reply:
This is referencing the pipeline.py variable thats is loaded with the phone numbers.